lib/Onyx.js

```diff
@@ -662,9 +664,10 @@ function getCollectionDataAndSendAsObject(matchingKeys, mapping) {
  *  component
  * @param {Boolean} [mapping.waitForCollectionCallback] If set to true, it will return the entire collection to the callback as a single object
  * @param {String|Function} [mapping.selector] THIS PARAM IS ONLY USED WITH withOnyx(). If included, this will be used to subscribe to a subset of an Onyx key's data.
- *       If the selector is a string, the selector is passed to lodashGet on the sourceData. If the selector is a function, the sourceData is passed to the selector and should return the
- *       simplified data. Using this setting on `withOnyx` can have very positive performance benefits because the component will only re-render when the subset of data changes.
- *       Otherwise, any change of data on any property would normally cause the component to re-render (and that can be expensive from a performance standpoint).
+ *       If the selector is a string, the selector is passed to lodashGet on the sourceData. If the selector is a function, the sourceData and withOnyx instance props are
```

Comment:
Hey @tgolen this comment/description also exists inside [API.md](https://github.com/Expensify/react-native-onyx/blob/58bc7cc22947f87f594edfedd8f0fbcccec8b30c/API.md?plain=1&#L131). Should we update this description there too?